frontend/src/pages/question/panels/NewQuestionPanelModal.jsx

```diff
@@ -14,13 +14,14 @@ import './panels.css';
  * @param {Object} panelStore new question panel custom hook
  */
 export default function NewQuestionPanelModal({ panelStore, onQuestionUpdated }) {
+  const concepts = useContext(ConceptsContext);
   /**
    * Get the panel background color
    * @param {Boolean} isNodePanel is panel of type node
    */
   function getBackgroundColor(isNodePanel) {
     // set the color of the node/edge panel header
-    const panelColorMap = getNodeTypeColorMap(config.concepts);
+    const panelColorMap = getNodeTypeColorMap(concepts);
```

Comment:
These shouldn't be gray right?
![Screen Shot 2020-12-23 at 9 30 17 AM](https://user-images.githubusercontent.com/1634866/103008151-a840c400-4502-11eb-8970-2c921d7bf9f1.png)

Reply:
They should. We have a hard-coded index of types to colors, and any type not in that index will be grey. If we want those to have colors, we just need to add them to the index and give them colors, but that should be dealt with in a separate PR.